arviz/plots/densityplot.py

```diff
@@ -44,23 +44,23 @@ def plot_density(
 
     Parameters
     ----------
-    data : Union[Object, Iterator[Object]]
+    data : object or iterable of object
         Any object that can be converted to an :class:`arviz.InferenceData` object, or an Iterator
         returning a sequence of such objects.
         Refer to documentation of :func:`arviz.convert_to_dataset` for details about such objects.
-    group: Optional[str]
+    group : str, optional
         Specifies which :class:`arviz.InferenceData` group should be plotted.
         Defaults to 'posterior'.
         Alternative values include 'prior' and any other strings used as dataset keys in the
         :class:`arviz.InferenceData`.
-    data_labels : Optional[List[str]]
+    data_labels : str, optional
```

Comment:
```suggestion
    data_labels : list of str, optional
```

`List[str]` indicates `list of str`